sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/BoundingRegion.java

```diff
@@ -41,22 +41,26 @@ void setPageNumber(int pageNumber) {
     }
 
     /**
-     * Get the boundingBox property: Bounding box on the page, or the entire page if not specified.
+     * Get the list of coordinates of bounding polygon, or the entire page if not specified.
+     * The numbers represent the x, y values of the polygon vertices, clockwise from the left (-180 degrees inclusive)
+     * relative to the element orientation.
      *
-     * @return the boundingBox value.
+     * @return the boundingPolygon value.
      */
-    public List<Float> getBoundingBox() {
-        return this.boundingBox;
+    public List<Point> getBoundingPolygon() {
```

Comment:
I think we just called this `polygon` in JS. The `bounding` nature is sort of implied already, especially in this instance since it's inside a class named `BoundingRegion`.